docs/src/configuration/app/hooks/hooks-comparison.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Comparison of hooks
+description: What hooks are available in the build and deploy process and how to choose which to use.
+---
+
+The following table presents the main differences among the three available hooks:
+
+| Hook name     | Failures stop build | Variables available | Services available | Run on `worker` instances | Writable directories | Blocks requests | Runs on all redeploys\* |
+| ------------- | ------------------- |------------------- | ------------------ | ------------------------- | -------------------- | --------------- | --------------- |
+| `build`       | Yes                 | Build variables     | No                 | Yes                       | `$PLATFORM_APP_DIR`, `$PLATFORM_CACHE_DIR`, and `/tmp` | No  | No  |
```

Comment:
Because the inline code doesn't break, it's almost the same as a bullet list. I'm just wary of adding more space into an already wide table and a bullet list will take up more room. :thinking: